modules/flatbuffers/24.3.7/MODULE.bazel

```diff
@@ -0,0 +1,73 @@
+module(
+    name = "flatbuffers",
+    version = "24.3.7",
+    compatibility_level = 1,
+    repo_name = "com_github_google_flatbuffers",
+)
+
+bazel_dep(
+    name = "aspect_bazel_lib",
+    version = "1.40.0",
+)
+bazel_dep(
+    name = "aspect_rules_esbuild",
+    version = "0.15.0",
+)
+bazel_dep(
+    name = "aspect_rules_js",
+    version = "1.34.1",
+)
+bazel_dep(
+    name = "aspect_rules_ts",
+    version = "1.4.5",
+)
+bazel_dep(
+    name = "grpc",
+    version = "1.48.1",
+    repo_name = "com_github_grpc_grpc",
+)
+bazel_dep(
+    name = "platforms",
+    version = "0.0.7",
+)
+bazel_dep(
+    name = "rules_cc",
+    version = "0.0.9",
+)
+bazel_dep(
+    name = "rules_go",
+    version = "0.41.0",
+    repo_name = "io_bazel_rules_go",
+)
+bazel_dep(
+    name = "rules_nodejs",
+    version = "5.8.3",
+)
+bazel_dep(
+    name = "rules_swift",
+    version = "1.2.0",
+    repo_name = "build_bazel_rules_swift",
+)
+# -- bazel_dep definitions -- #
+
+npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+npm.npm_translate_lock(
+    name = "npm",
+    npmrc = "//:.npmrc",
+    pnpm_lock = "//:pnpm-lock.yaml",
+    verify_node_modules_ignored = "//:.bazelignore",
+)
+use_repo(npm, "npm")
+
+node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
+use_repo(node, "nodejs_linux_amd64")
+
+rules_ts_ext = use_extension("@aspect_rules_ts//ts:extensions.bzl", "ext", dev_dependency = True)
+rules_ts_ext.deps()
+use_repo(rules_ts_ext, "npm_typescript")
+
+non_module_dependencies = use_extension("//:extensions.bzl", "non_module_dependencies")
+use_repo(
+    non_module_dependencies,
```

Comment:
Why do we need a Bazel binary as dependency? Is it only a dev dependency for tests?

Reply:
Yeah, let's try that.